cranelift/codegen/src/machinst/abi.rs

```diff
@@ -927,6 +902,23 @@ impl SigSet {
 
         clobbers
     }
+
+    /// Get the number of arguments expected.
+    pub fn num_args(&self, sig: Sig) -> usize {
+        let sig_data = &self.sigs[sig];
+        let num = if sig_data.stack_ret_arg.is_some() {
+            self.args(sig).len() - 1
+        } else {
+            self.args(sig).len()
+        };
+
+        usize::from(num)
+    }
+
+    /// Get the number of return values expected.
+    pub fn num_rets(&self, sig: Sig) -> usize {
+        usize::from(self.rets(sig).len())
```

Comment:
The result of calling `slice::len` is a `usize`, so this call to `usize::from` isn't necessary in this version.